### DISCUSSION TITLE
site-to-site vpn

### DISCUSSION THREAD

**Discussion by hwz9612** [strongswan/strongswan#2258] [Q&A]
Hi,
Based on the topology shown below, I establish a site-to-site VPN.
![image](https://github.com/strongswan/strongswan/assets/35334392/4a40a2a1-ef50-4b08-8510-951f0ed6de4e)
The configurations for the Sun host are as follows:
![image](https://github.com/strongswan/strongswan/assets/35334392/f77b61a4-203a-4d85-9bd6-0996ee873fd6)
![image](https://github.com/strongswan/strongswan/assets/35334392/a7826203-769b-49ea-976a-0df202baaac5)
The configurations for the Moon host are as follows:
![image](https://github.com/strongswan/strongswan/assets/35334392/e0396150-3de7-4da5-b326-fcc053a604ff)
![image](https://github.com/strongswan/strongswan/assets/35334392/11a6ba76-7351-4bbb-9f24-969ce740fee4)
When I ping Moon host from the Sun host, 
![image](https://github.com/strongswan/strongswan/assets/35334392/0176632b-bb7d-44a6-84ca-6e7104979e82)
But I ping Host2 from Host1, it can not echo it.
![image](https://github.com/strongswan/strongswan/assets/35334392/e855f97a-fd79-4a12-85a2-f459380bdffe)
What should I do?

**Comment by tobiasbrunner** [comment]
Is there a reason for that host to send packets addressed to 10.2.0.9 to sun so they could get encrypted? If 10.1.0.8 is not the default gateway in that subnet, host1 will require a custom route for 10.2.0.0/24 via sun. Similarly, on the other end for host2 and moon. Also, forwarding has obviously to be enabled on the gateways (and the firewall rules have to allow the traffic etc.), see [the docs](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html).

**Comment by hwz9612** [comment]
Hi,
I have solved this problem. But I met a new problem. Now,
![image](https://github.com/strongswan/strongswan/assets/35334392/6d96a495-6f1c-47dc-8388-371058b72b9d)
According to the new topology, I build the tunnel between the sun and moon. Then I build the tunnel between sun1 and moon1. I met the following problem.
![image](https://github.com/strongswan/strongswan/assets/35334392/de5eb92c-3838-4e98-b346-4346086b7a82)
What should I do?

**Reply by tobiasbrunner** [reply]
Read the log. It should tell you why that notify was sent (try increasing the log level, see #196).

**Reply by hwz9612** [reply]
When I executed the following command, it echo:
![image](https://github.com/strongswan/strongswan/assets/35334392/dd338c9c-139a-4604-8108-dcf1199a56e1)
Please see the attached log for details.
[charon.log](https://github.com/strongswan/strongswan/files/15413519/charon.log)
